src/styles/components/_modal.scss

```diff
@@ -83,7 +83,6 @@
   }
 
   &__body--open {
-    position: fixed;
```

Comment:
actually we need the position fixed, it's the overflow we can remove

Reply:
also, it seems this is not only impacting mobile safari, but also web and mobile chrome (check the bottom of the page when modal is open).  I'm not sure why we have overflow auto there originally, but I haven't had any problems with it removed on safari and chrome (gonna go run off and check FF now...)

Reply:
confirmed, overflow is safely removed on FF also

Reply:
Fixed positioning was originally to prevent a double scrollbar (I think) on pages where the modal content exceeds the height of the viewport. Fixed positioning makes the body content fit within 100vh so the only scrolling that would happen would be inside of the modal. 

This might not affect this particular experiment / modal, but you should test / verify on modals with content that overflows the window since this modal style affects every modal.